Cabal/Distribution/PackageDescription/Check.hs

```diff
@@ -36,6 +36,7 @@ module Distribution.PackageDescription.Check (
 import Data.Maybe
          ( isNothing, isJust, catMaybes, maybeToList, fromMaybe )
 import Data.List  (sort, group, isPrefixOf, nub, find)
+import Control.Applicative
```

Comment:
```
Distribution/PackageDescription/Check.hs:39:1: Warning:
    The import of ‘Control.Applicative’ is redundant
      except perhaps to import instances from ‘Control.Applicative’
    To import instances alone, use: import Control.Applicative()
```

assuming `Control.Applicative` is really needed, you can also just add an `import Prelude` after all `base`-imports